README.md

```diff
@@ -69,9 +69,10 @@ Each new policy should contain:
 * `kustomization.yaml` and `suite.yaml` under `library/<policy-name>`
 * The rego source, as `src.rego` and unit tests as `src_test.rego` in the corresponding subdirectory under `src/<policy-name>`
 * [Versioning](https://docs.google.com/document/d/1IYiypA-mRcdfSVfmoeyuaeG8XtA1u4GkcqH3kEkv2uw/edit) has been introduced for Gatekeeper Library policies. Please make sure to add or bump the version of the policy as per the guidelines in the `src/<policy-name>/constraint.tmpl` annotation.
-  * Major version bump required: Whenever there is a breaking change in the policy e.g.  updating template Kind, parameter schema, or any other breaking changes
+  * Major version bump required: Whenever there is a breaking change in the policy e.g.  updating template Kind, existing parameter schema, adding the `requires-sync-data` annotation, or any other breaking changes
```

Comment:
If the template always required sync data and we are just adding the `requires-sync-data` annotation because it was omitted, should that be a version bump?

Reply:
IMO, if sync data was already required, then adding the annotation is not a breaking change

Reply:
Agreed - I was thinking new because there was a new sync data requirement.